puzzleAI/settings.py

```diff
@@ -0,0 +1,115 @@
+from pathlib import Path
+
+from my_settings import DATABASES, SECRET_KEY
+
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+DEBUG = True
+
+ALLOWED_HOSTS = ['*']
+
+INSTALLED_APPS = [
+    'django.contrib.admin',
+    'django.contrib.auth',
+    'django.contrib.contenttypes',
+    'django.contrib.sessions',
+    'django.contrib.messages',
+    'django.contrib.staticfiles',
+    'corsheaders',
+    'rest_framework',    
+]
+
+MIDDLEWARE = [
+    'django.middleware.security.SecurityMiddleware',
+    'django.contrib.sessions.middleware.SessionMiddleware',
+    'django.middleware.common.CommonMiddleware',
+    'django.middleware.csrf.CsrfViewMiddleware',
+    'django.contrib.auth.middleware.AuthenticationMiddleware',
+    'django.contrib.messages.middleware.MessageMiddleware',
+    'django.middleware.clickjacking.XFrameOptionsMiddleware',
+    'corsheaders.middleware.CorsMiddleware',
```

Comment:
`corsheaders`가 올바르게 작동하기 위해선 `MIDDLEWARE` 안에서 `corsheaders`의 위치가 중요합니다.
[링크](https://pypi.org/project/django-cors-headers/)를 참고하시면 좋을 것 같습니다!

> CorsMiddleware should be placed as high as possible, especially before any middleware that can generate responses such as Django’s CommonMiddleware or Whitenoise’s WhiteNoiseMiddleware. If it is not before, it will not be able to add the CORS headers to these responses.

Reply:
지연님 피드백 감사 드립니다. 해당 내용 반영하겠습니다.